.github/workflows/ci.yml

```
@@ -125,6 +125,40 @@ jobs:
+  # uses its own instrumentation flags, so it does not share the test job's
+  # cache. Fork pull requests have no secret and upload tokenless. A failed
+  # upload never turns the job red: coverage is a report, not a gate.
+  coverage:
+    name: Coverage
+    runs-on: ubuntu-latest
+    needs: prepare
+    env:
+      RUSTFLAGS: "-C linker-features=-lld"
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions-rs/toolchain@v1
```

actionlint (1.7.12)

[error] the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)